pulsar-common/src/main/java/org/apache/pulsar/common/schema/GetSchemaResponse.java

```diff
@@ -32,6 +32,6 @@
     private long version;
     private SchemaType type;
     private long timestamp;
-    private String data;
+    private byte[] data;
```

Comment:
This has to be in a new version of rest endpoint. Otherwise it is a BC problem.

Reply:
I have reduced the scope of this pr to just a). the b) part should be in another pr